src/main/java/com/alkemy/ong/service/CategoryService.java

```diff
@@ -78,4 +85,25 @@ public CategoryDtoFull getCategory(String id) {
                     "Category not found");
         }
     }
+    //Get categories for pages
+    @Transactional
+    public PagesDto<CategoryDto> getAllForPages(int page) {
+        if (page < 0) {
+            throw new ParamNotFound("The page number cannot be less than 0.");
+        }
+        Pageable pageRequest = PageRequest.of(page, 10);
+        Page<CategoryEntity> category = categoryRepository.findAll(pageRequest);
```

Comment:
I also added a constant PAGE_SIZE, so it is easier to change the number of entities in each page